client/network/bitswap/src/lib.rs

```diff
@@ -470,12 +470,14 @@ mod tests {
 		let mut client = TestClientBuilder::with_tx_storage(u32::MAX).build();
 		let mut block_builder = client.new_block(Default::default()).unwrap();
 
-		let ext = Extrinsic::Store(vec![0x13, 0x37, 0x13, 0x38]);
+		// encoded extrsinic: [161, .. , 2, 6, 16, 19, 55, 19, 56]
+		let ext = UncheckedExtrinsicBuilder::new_store(vec![0x13, 0x37, 0x13, 0x38]).build();
+		let pattern_index = ext.encoded_size() - 4;
```

Comment:
The test is looking for some pattern:  `0x13, 0x37, 0x13, 0x38` (`19,55,19,56`).
Previously this pattern was at index 2 of scale-encoded extrinsic, (simply enum carrying vec [6, 16, 19, 55, 19, 56]).

Now extrinsic was replaced with more complicated `UncheckedExtrinsic`. Instead of computing the index (at which the pattern would be located in new layout) from the beggining, it is now computed it from the end. 

The pattern will always be at the end of the scale-encoded `UncheckedExtrinsic` containing `pallet::Call::store`.

The full encoded buffer:
```
[161, 1, 132, 212, 53, 147, 199, 21, 253, 211, 28, 97, 20, 26, 189, 4, 169, 159, 214, 130, 44, 133, 88, 133, 76, 205, 227, 154, 86, 132, 231, 165, 109, 162, 125, 162, 177, 132, 10, 137, 58, 75, 135, 50, 68, 163, 254, 88, 113, 214, 53, 129, 184, 18, 98, 65, 7, 41, 76, 229, 21, 155, 39, 213, 77, 143, 115, 90, 139, 70, 129, 231, 130, 232, 178, 12, 155, 20, 97, 243, 111, 244, 17, 34, 200, 123, 2, 136, 9, 44, 106, 143, 182, 214, 196, 200, 243, 139, 132, 2, 6, 16, 19, 55, 19, 56]
```

Reply:
Got it. Thank you